v2x/vlog_to_model.py

```diff
@@ -20,6 +20,14 @@
     - `(* ASSOC_CLOCK="RDCLK" *)` : force a port's associated
                                     clock to a given value
 
+    - `(* NO_COMB=1 *)` : Forces removal of all combinational relations of an
+                          input port.
+
+    - `(* NO_COMB=0 *)` : Allows a clock port to have combinational sinks
```

Comment:
Make sure there is a test for this.

Reply:
There is a test under `tests/clock_mux` that verifies that.